content/docs/installation/operator-lifecycle-manager.md

```diff
@@ -91,9 +91,146 @@ in accordance with [OLM's Recommended Channel Naming][].
 If you have any issues with your installation, please refer to the
 [FAQ](../faq/README.md).
 
+## Configuration
+
+The configuration options are quite limited when you install cert-manager using OLM.
```

Comment:
Just out of curiosity: are the options limited because of limitations with OLM or is it limitation we have imposed?

Reply:
The current version of OLM has limited support for modifying the deployment, as documented in this PR. 
For example, it does not provide an easy way for the OLM package author to allow the container command line flags to be overridden. 
But on the other hand, if all the cert-manager components could be configured using a config file in a configmap and if all the existing command line options could also be declared inthose config files, that particular problem would be solved.

There are some other limitations such as not being able to customize the resource requests of individual Deployments in the OLM package:
 * https://github.com/operator-framework/operator-lifecycle-manager/issues/1794

The current API seems to be designed around there only being a single Deployment in each package, but cert-manager has three.